### DISCUSSION TITLE
Running on GPU?

### DISCUSSION THREAD

**Discussion by ghost** [ContinualAI/avalanche#982] [Q&A] [answered]
Hi everybody

How can I know if the experiments are running in the CPU or GPU? If by default is the CPU, how can I change it to run them in the GPU? 

```
print(torch.cuda.is_available())
True
```

Thank you in advance

**Comment by AntonioCarta** [comment, marked as answer]
Strategies have an argument `device`. Example:
```
strat = Naive(..., device='cuda')  # model will run on GPU
```
To verify that the model is actually using the GPU I would just check nvidia-smi.
